scripts/db-advanced-test.py

```
@@ -0,0 +1,351 @@
+"""
+
+import os
+import sys
+import random
+import uuid
+import json
+from datetime import datetime, timedelta
+import psycopg2
+from psycopg2.extras import RealDictCursor
+import logging
+from typing import Dict, List, Any, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)